src/Enterspeed.Source.UmbracoCms.V9/Guards/ContentCultureUrlRequiredGuard.cs

```diff
@@ -12,7 +13,7 @@ public bool CanIngest(IPublishedContent content, string culture)
                 return true;
             }
 
-            var url = content.Url(culture);
+            var url = content.GetUrl(culture);
             return !string.IsNullOrWhiteSpace(url) && !url.Equals("#");
```

Comment:
@elglogins Could it make sense to log the url and culture variable? When debugging this we had to step through the code. If we could have seen this in the logs, it might have been faster to spot the issue from the updated Umbraco?

Reply:
One level higher where this is called there is a log message in form of:
```
"Content {contentId} with {culture} culture, ingest avoided by '{guard}'."
```

Reply:
Yeah, we found that (after increasing the debug level), I was thinking on the URL. But perhaps it's just a freak incident?

Reply:
Added